assets/xml/objects/object_fish.xml

```diff
@@ -1,52 +1,52 @@
-        <Limb Name="gFishingFishLeftPectoralFinLimb" Type="Standard" EnumName="FISHING_FISH_LIMB_LEFT_PECTORAL_FIN" Offset="0x2978" />
-        <Skeleton Name="gFishingFishSkel" Type="Flex" LimbType="Standard" LimbNone="FISHING_FISH_LIMB_NONE" LimbMax="FISHING_FISH_LIMB_MAX" EnumName="FishingFishLimb" Offset="0x29C0" />
+        <!-- Bass -->
+
+        <!-- Bass Animations -->
+        <Animation Name="gFishingBassAnim" Offset="0x7C" /> <!-- Original name is "basu" -->
+
+        <!-- Bass DLs -->
+        <DList Name="gFishingBassTailFinDL" Offset="0x940" />
+        <DList Name="gFishingBassBackSegmentTipDL" Offset="0x9E8" />
+        <DList Name="gFishingBassTopRearFinDL" Offset="0xB00" />
+        <DList Name="gFishingBassBottomRearFinDL" Offset="0xBA0" />
+        <DList Name="gFishingBassBackSegmentDL" Offset="0xC40" />
+        <DList Name="gFishingBassTopFrontFinDL" Offset="0xE30" />
+        <DList Name="gFishingBassBottomFrontFinDL" Offset="0xED0" />
+        <DList Name="gFishingBassMiddleSegmentDL" Offset="0xF70" />
+        <DList Name="gFishingBassHeadDL" Offset="0x10A0" />
+        <DList Name="gFishingBassLeftPectoralFinDL" Offset="0x1260" />
+        <DList Name="gFishingBassRightPectoralFinDL" Offset="0x1300" />
+        <DList Name="gFishingBassJawDL" Offset="0x13A0" />
+
+        <!-- Bass Textures -->
+        <Texture Name="gFishingBassTailFinTex" OutName="fishing_bass_caudal_fin" Format="rgba16" Width="16" Height="16" Offset="0x14D0" />
+        <Texture Name="gFishingBassTopScalesTex" OutName="fishing_bass_dorsal_scales" Format="rgba16" Width="16" Height="16" Offset="0x16D0" />
+        <Texture Name="gFishingBassBottomScalesTex" OutName="fishing_bass_belly_scales" Format="rgba16" Width="16" Height="16" Offset="0x18D0" />
+        <Texture Name="gFishingBassHeadTex" OutName="fishing_bass_head" Format="rgba16" Width="32" Height="32" Offset="0x1AD0" />
+        <Texture Name="gFishingBassFinTex" OutName="fishing_bass_fin" Format="rgba16" Width="16" Height="16" Offset="0x22D0" />
+        <Texture Name="gFishingBassUnusedFinTex" OutName="fishing_bass_unused_fin" Format="rgba16" Width="16" Height="16" Offset="0x24D0" />
+        <Texture Name="gFishingBassSpinyFinTex" OutName="fishing_bass_spiny_fin" Format="rgba16" Width="16" Height="16" Offset="0x26D0" />
+
+        <!-- Bass Limbs and Skel -->
+        <Limb Name="gFishingBassHeadLimb" Type="Standard" EnumName="FISHING_BASS_LIMB_HEAD" Offset="0x28D0" />
+        <Limb Name="gFishingBassMiddleSegmentRootLimb" Type="Standard" EnumName="FISHING_BASS_LIMB_MIDDLE_SEGMENT_ROOT" Offset="0x28DC" />
+        <Limb Name="gFishingBassBackSegmentRootLimb" Type="Standard" EnumName="FISHING_BASS_LIMB_BACK_SEGMENT_ROOT" Offset="0x28E8" />
+        <Limb Name="gFishingBassTipRootLimb" Type="Standard" EnumName="FISHING_BASS_LIMB_END_ROOT" Offset="0x28F4" />
```

Comment:
Looks like there's a mismatch between the Limb and EnumName here. Honestly, `gFishingBassEndRootLimb` is fine with me if you prefer that.

Reply:
augh sorry i kinda rushed it yeah ill revert it to `EndRoot`
alsoi need to remember to build before committing